src/open_inwoner/components/templates/components/Pagination/Pagination.html

```diff
@@ -71,17 +71,17 @@
         {% endfor %}
 
         {% if page_obj.has_next %}
-            <a class="pagination__link" 
+            <a class="pagination__next pagination__link"
                 {% if hxget %}
                     hx-get="{{ hxget }}?{% querystring key='page' value=page_obj.next_page_number %}" hx-trigger="click" hx-target="{{ hxtarget }}"
                 {% else %}
                     href="?{% querystring key='page' value=page_obj.next_page_number %}"
                 {% endif %}
             >
-                {% icon icon="chevron_right" %}
+                {% icon icon="east" %}
             </a>
         {% else %}
-            <span class="pagination__item">{% icon icon="chevron_right" %}</span>
+            <span class="pagination__item">{% icon icon="east" %}</span>
```

Comment:
If I click on the last page, the page number collides with the arrow: 
![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/af0510fa-23ce-4564-8b74-bdad53e6c7c5)

Reply:
Well spotted, fixed it now.